### PR TITLE
Latest files from Crowdin

### PR DESCRIPTION
**Pull request by zapbot**
Autogenerated PR from zapbot

### CODE REVIEW

File: src/org/zaproxy/zap/extension/selenium/resources/Messages_zh_CN.properties [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #X-Generator: crowdin.com
 
 selenium.extension.ui.name=Webdriver\u7684\u63d0\u4f9b\u7a0b\u5e8f
-selenium.extension.desc=Provides WebDrivers to control several browsers using Selenium and includes HtmlUnit browser.
+selenium.extension.desc=Bevat WebDrivers om verschillende browsers te besturen met Selenium en bevat HtmlUnit browser.
```
Comment (kingthorin) [inline src/org/zaproxy/zap/extension/selenium/resources/Messages_zh_CN.properties]: Is this German in the Chinese help?

File: src/org/zaproxy/zap/extension/gettingStarted/resources/Messages_zh_CN.properties [hunk 2]
```diff
@@ -4,4 +4,4 @@
 gettingStarted.desc=ZAP\u5165\u95e8\u6307\u5357
 gettingStarted.menu=\u83b7\u53d6\u5f00\u59cb\u7d22\u5f15
 # gettingStarted.file is the filename - do NOT translate it unless you have also provided a translated file with that name 
-gettingStarted.file=ZAPGettingStartedGuide-2.6.pdf
+gettingStarted.file=ZAPGettingStartedGuide-2.4.pdf
```
Comment (kingthorin) [inline src/org/zaproxy/zap/extension/gettingStarted/resources/Messages_zh_CN.properties]: ?

Comment (kingthorin) [inline src/org/zaproxy/zap/extension/gettingStarted/resources/Messages_zh_CN.properties]: ?

File: src/org/zaproxy/zap/extension/selenium/resources/Messages_zh_CN.properties [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
```
Comment (kingthorin) [inline src/org/zaproxy/zap/extension/selenium/resources/Messages_zh_CN.properties]: Is this German in the Chinese help?

File: src/org/zaproxy/zap/extension/gettingStarted/resources/Messages_zh_CN.properties (hunk 2 above)
Comment (psiinon) [inline src/org/zaproxy/zap/extension/gettingStarted/resources/Messages_zh_CN.properties]: I've changed that on Crowdin, so is should be right next time...

File: src/org/zaproxy/zap/extension/selenium/resources/Messages_zh_CN.properties (hunk 3 above)
Comment (thc202) [inline src/org/zaproxy/zap/extension/selenium/resources/Messages_zh_CN.properties]: Yeah, a lot of mixed translations.

**Pull request closed without merging**